sormas-backend/src/main/java/de/symeda/sormas/backend/common/AbstractCoreAdoService.java

```diff
@@ -35,4 +50,63 @@ public boolean isArchived(String uuid) {
 		long count = em.createQuery(cq).getSingleResult();
 		return count > 0;
 	}
+
+	protected <T extends ChangeDateBuilder<T>> T addChangeDates(T builder, From<?, ADO> adoPath, boolean includeExtendedChangeDateFilters){
+		return builder.add(adoPath);
+	}
+
+	public Date calculateCaseEndOfProcessingDate(List<String> entityuuids) {
+
+		if (entityuuids.isEmpty()) {
+			return null;
+		}
+		CriteriaBuilder cb = em.getCriteriaBuilder();
+		CriteriaQuery<Date> cq = cb.createQuery(Date.class);
+		Root<ADO> from = cq.from(getElementClass());
+
+		Expression agregatedChangeDateExpression = addChangeDates(new AgregatedChangeDateExpressionBuilder(cb), from, true).build();
```

Comment:
```suggestion
		Expression aggregatedChangeDateExpression = addChangeDates(new AgregatedChangeDateExpressionBuilder(cb), from, true).build();
```

Reply:
update the variable name along with the class name